library/cc/stream_callbacks.cc

```diff
@@ -79,15 +79,15 @@ void* EnvoyHttpCallbacksAdapter::c_on_error(envoy_error raw_error, void* context
 void* EnvoyHttpCallbacksAdapter::c_on_complete(void* context) {
   auto self = static_cast<EnvoyHttpCallbacksAdapter*>(context);
   if (self->stream_callbacks_->on_complete.has_value()) {
-    self->stream_callbacks_->on_complete.value();
+    self->stream_callbacks_->on_complete.value()();
```

Comment:
nit: this is a little bit wacky. If nothing else, maybe pull into a local variable first.